src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java

```diff
@@ -106,11 +106,11 @@ public void testJavadocMethod() throws Exception {
     }
 
     @Test
+    @Ignore("until https://github.com/checkstyle/patch-filters/issues/136")
```

Comment:
Please explain in the issue description what problem we have and why this test is ignored. As I understand it is caused by missing comment node in AST. If this issue is common for multiple checks please create one common github issue, don't need to create same issue for every check.

Reply:
done, https://github.com/checkstyle/patch-filters/pull/137#issue-455742771